src/reducers/status/reducers.js

```diff
@@ -0,0 +1,48 @@
+import { calculateWinner } from '../../components/game/controller';
```

Comment:
Call this file: `reducer.js` as opposed to `reducers.js`. Only one reducer currently exists in this file.